watertap/core/util/tests/test_initialization.py

```diff
@@ -1,5 +1,5 @@
 #################################################################################
-# WaterTAP Copyright (c) 2020-2024, The Regents of the University of California,
+# WaterTAP Copyright (c) 2020-2025, The Regents of the University of California,
```

Comment:
@adam-a-a I have been curious about when we were going to address this issue in main (I have been doing it in REFLO all year). Should it just be a "change as you go" type thing?

doesn't really matter for this PR but the change caught my eye.